xwiki-platform-core/xwiki-platform-comment/xwiki-platform-comment-default/src/main/java/org/xwiki/platform/comment/internal/DefaultComment.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.platform.comment.internal;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.xwiki.platform.comment.Comment;
+import org.xwiki.platform.comment.CommentId;
+import org.xwiki.user.UserReference;
+
+public class DefaultComment implements Comment
```

Comment:
Feels odd to see a single word role name. We usually have a suffix. Like CommentManager

Reply:
Probably explained by the fact that this is a POJO. Shouldn't it just be one instead of a component?

Reply:
Not sure what you mean here, it's not a component...

Reply:
Indeed. Though I still fail to see the need for a split between interface and class here.